File: JavaScript projects/JavaScript Basics/03.Conditional Statements Advanced/Exercises/02_summer_outfit.js
function summerOutfit(degree, daytime){
    let outfit;
    let shoes;
    if (10 <= degree && degree <= 18){
        switch(daytime){
            case "Morning":
                outfit = "Sweatshirt";
                shoes = "Sneakers";
                break;
            case "Afternoon":
                outfit = "Shirt";
                shoes = "Moccasins";
                break;
            case "Evening":
                outfit = "Shirt";
                shoes = "Moccasins";
                break;
        }
    }else if (18 < degree && degree <= 24){
        switch(daytime){
            case "Morning":
                outfit = "Shirt";
                shoes = "Moccasins";
                break;
            case "Afternoon":
                outfit = "T-Shirt";
                shoes = "Sandals";
                break;
            case "Evening":
                outfit = "Shirt";
                shoes = "Moccasins";
                break;
        }
    }else{
        switch(daytime){
            case "Morning":
                outfit = "T-Shirt";
                shoes = "Sandals";
                break;
            case "Afternoon":
                outfit = "Swim Suit";
                shoes = "Barefoot";
                break;
            case "Evening":
                outfit = "Shirt";
                shoes = "Moccasins";
                break;
        }
    }
    console.log(`It's ${degree} degrees, get your ${outfit} and ${shoes}.`);
}
summerOutfit(16,"Morning");

File: Python projects/Python Basics/03_conditional_statements_advanced/exercises/02_summer_outfit.py
degree = int(input())
daytime = str(input())
outfit = ''  # outfit and shoes could be set to shirt and moccasins since they occur 5 times out of 9
shoes = ''   # this would result in far less entries, but this way it is easier to follow the purpose of the exercise

if 10 <= degree <= 18:
    if daytime == "Morning":
        outfit = 'Sweatshirt'
        shoes = 'Sneakers'
    elif daytime == "Afternoon":
        outfit = 'Shirt'
        shoes = 'Moccasins'
    else:
        outfit = 'Shirt'
        shoes = 'Moccasins'
elif 18 < degree <= 24:
    if daytime == "Morning":
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif daytime == "Afternoon":
        outfit = 'T-Shirt'
        shoes = 'Sandals'
    else:
        outfit = 'Shirt'
        shoes = 'Moccasins'
else:
    if daytime == "Morning":
        outfit = 'T-Shirt'
        shoes = 'Sandals'
    elif daytime == "Afternoon":
        outfit = 'Swim Suit'
        shoes = 'Barefoot'
    else:
        outfit = 'Shirt'
        shoes = 'Moccasins'

print(F"It's {degree} degrees, get your {outfit} and {shoes}.")
